airbyte/sources/util.py

```diff
@@ -116,6 +117,51 @@ def get_source(  # noqa: PLR0913 # Too many arguments
     )
 
 
+def get_benchmark_source(
+    num_records: int | str = 1000,
+) -> Source:
```

Comment:
@coderabbitai - I considered calling this `get_dummy_source` or something else. Any suggestions of synonyms or other names that would be easily understandable. This function is a good test for performance and also for testing basic functionality in destinations.

Reply:
@coderabbitai - I like `mock` and `dummy`. I don't love `synthetic` because it feels verbose.

I think `benchmark` is probably best for the current code signature and goals for this iteration. In the future, we might go more generic by adding `get_dummy_source` or `get_mock_source` but in those cases, our function signature might expose additional arguments that would be more generic. In that case, `get_benchmark_source` might wrap the "dummy" or "mock" version with specific parameters. 

So, I lean towards keeping the name as-is, but holding space for future iterations with a more generic function signature. Wdyt?